app/controllers/insured/families_controller.rb

```diff
@@ -698,6 +698,8 @@ def resolve_layout
       EnrollRegistry.feature_enabled?(:bs4_consumer_flow) ? "progress" : "application"
     when "purchase"
       "application"
+    when "verification"
+      return "progress" if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
```

Comment:
need to add the ternary, need to specify which layout it uses if it isn't enabled